github/files/drupal7/scripts/devops/reinstall/pre_settings.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Adding database variable to settings.php
   sudo: yes
-  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"127.0.0.1\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
+  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"{{ mysql_host }}\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
```

Comment:
@danylevskyi Yep, help me please ) because I've not any ideas, why a `mysql_host` is hard coded to '127.0.0.1' value throughout the code and it should be changed...

P.S. It is hard to understand for me that add a `login_host` parameter to `mysql_db` ansible module is a bad practice.

/cc @podarok